inai-ios-sample-integration.xcodeproj/project.pbxproj

```diff
@@ -44,8 +44,19 @@
 		28F2D4A5289301DB007EF5B3 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 28F2D4A3289301DB007EF5B3 /* Main.storyboard */; };
 		28F2D4A7289301DD007EF5B3 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 28F2D4A6289301DD007EF5B3 /* Assets.xcassets */; };
 		28F2D4AA289301DD007EF5B3 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 28F2D4A8289301DD007EF5B3 /* LaunchScreen.storyboard */; };
-		45ACC28367AAF145DA7198A6 /* Pods_inai_ios_sample_integration.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 7EC009725747D5E7A3279F62 /* Pods_inai_ios_sample_integration.framework */; };
+		610AD66003DDCE30DC344D7F /* Pods_inai_ios_sample_integration.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 7EC009725747D5E7A3279F62 /* Pods_inai_ios_sample_integration.framework */; };
```

Comment:
Do we need these files to be pushed?

Reply:
This file contains all the changes entire project we need this file